test/spec/java_spec.rb

```diff
@@ -8,7 +8,7 @@
 LATEST_ZULU_OPENJDK_11_STRING = 'OpenJDK Runtime Environment Zulu11.78+15-CA (build 11.0.26+4-LTS)'
 LATEST_ZULU_OPENJDK_17_STRING = 'OpenJDK Runtime Environment Zulu17.56+15-CA (build 17.0.14+7-LTS)'
 LATEST_ZULU_OPENJDK_21_STRING = 'OpenJDK Runtime Environment Zulu21.40+17-CA (build 21.0.6+7-LTS)'
-LATEST_ZULU_OPENJDK_24_STRING = 'OpenJDK Runtime Environment Zulu24.28+87-CRaC-CA (build 24+36)'
+LATEST_ZULU_OPENJDK_24_STRING = 'OpenJDK Runtime Environment Zulu24.28+83-CA (build 24+36)'
```

Comment:
Since this buildpack now shares the distributions with the CNB and this release already existed in the new bucket, I chose to keep the CNB release instead of the classic one. This seems to be fine since:

a) Only 0.01% (sic!) of installs do use OpenJDK 24
b) The change removes `CRaC` which we do not support anyways (it was erroneously included)
c) The Azul build number has no direct customer impact and we never guaranteed a specific one